docs/pages/guides/_meta.js

```diff
@@ -1,7 +1,7 @@
 export default {
   "replicating-onchain-state": "Replicating onchain state",
   "hello-world": "Hello World",
-  "extending-a-world": "Extending a World",
+  extending: "Extending a World",
```

Comment:
Reminder to set up redirects if we’re changing URLs

Reply:
Oops, you're right. Fixed.